lib/Transforms/IPO/StructFieldCacheAnalysisImpl.h

```diff
@@ -0,0 +1,377 @@
+// lib/Tranforms/IPO/StructFieldCacheAnalysisImpl.h - Performs Cache-Aware
```

Comment:
Yes. That's what I mean. But I didn't name it as StructFieldCacheAnalysis.h because there's a header file with the same name in include/ and I don't want to put my class declarations in it to make it public...